src/ops/fixit.rs

```diff
@@ -41,6 +45,7 @@ impl FixitArgs {
 #[derive(Debug, Default)]
 struct File {
     fixes: u32,
+    original_code: String,
```

Comment:
Maybe `original_source`?

When reading about exit codes and original code, it can get easy to mix up what meaning of `code` is being used